src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -182,9 +182,10 @@ function BaseTextInput(
             }
 
             const layout = event.nativeEvent.layout;
+            const HEIGHT_TO_FIT_EMOJIS = 1;
```

Comment:
Can you leave a comment before this explaining what this is used for? Also, I don't understand why this is uppercase, as it was a global. Wouldn't it be better to define it in the CONST file?

Reply:
Updated! https://github.com/Expensify/App/pull/47547/commits/582d4ba49cd2038b92a0a37a1923848a371b8c3d